topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -0,0 +1,355 @@
+>    - {% icon param-collection %} *"Dataset collection"*: `soil collection processed` 
+>
+> 4. Rename the outputs as `FastQC processed: Raw` and `FastQC processed: Web` 
+> 3. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*:
+>
+>		- `FastQC processed: Raw`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+> 
+>
+{: .hands_on}
+
+We can verify that after processing the samples, the GC content presents a unimodal distribution, which indicates that the anomalies in the sequences have been successfully eliminated.
+
+![FasTQC plots CG](../../images/metagenomics-nanopore/fasqc_per_sequence_content_plot_clean.png "Per sequence CG content in processed samplesp")
+
+# Assign taxonomic classifications
+
+One of the key steps in metagenomic data analysis is to identify the taxon to which the individual reads belong. Taxonomic classification tools are based on microbial genome databases to identify the origin of each sequence.
+
+## Taxonomic classification with Kraken2
+
+In order to perform the taxonomic classification we will use __Kraken2__ ({% cite Wood_2019 %}), a tool which assign labels by examining the k-mers within a sequence and querying them in a database that uses a compact hash table for algorithm efficiency ({% cite deKoning2020 %}).
+
+![Taxonomic classification](../../images/metagenomics-nanopore/kmers-kraken.jpg "Kramer2 sequence classification algoritm ({% cite Wood2014 %})")
+
+For this tutorial we will use the __SILVA database__ ({% cite Quast2012 %}). It includes over 3.2 million 16S rRNA sequences from the _Bacteria_, _Archaea_ and _Eukaryota_ domains.
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. {% tool [Kraken2](toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.0.8_beta+galaxy0) %} with the following parameters:
+>    - *"Single or paired reads"*: `Single`
+>    - {% icon param-collection %} *"Dataset collection"*: `soil collection processed` 
+>    - *"Print scientific names instead of just taxids"*: `Yes`
+>    - *"Confidence"*: `0.1`
+>    - In *"Create Report"*:
+>        - *"Print a report with aggregrate counts/clade to file"*: `Yes`
+>        - *"Format report output like Kraken 1's kraken-mpa-report"*: `Yes`
+>    - *"Select a Kraken2 database"*: `Silva (Created: 2020-06-24T164526Z, kmer-len=35, minimizer-len=31, minimizer-spaces=6)`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > A confidence score of 0.1 means that at least 10% of  kmers should match with de databse.  This value can be reduced if a less restrictive taxonomic assignation is desired. 
+>    {: .comment}
+>
+{: .hands_on}
+
+
+> ### {% icon question %} Questions
+>
+> What does the term k-mer mean?
```

Comment:
isn't it too late for this? I have been wondering what a k-mer is for 20 lines or so ;)